Editor/Resources/EditorWindow/Components/DeploymentScenarios.uxml

```diff
@@ -34,8 +34,6 @@
                 </ui:VisualElement>
             </ui:VisualElement>
         </ui:VisualElement>
-        <ui:VisualElement class="form-row form-row--free form-row__input form-row__input--no-label">
-            <ui:Button name="ManagedEC2ScenarioShowMoreButton" class="button" text="Show more Options"/>
-        </ui:VisualElement>
+        <ui:Button name="ManagedEC2ScenarioShowMoreButton" class="form-row__input form-row__input--no-label button" text="Show more Options"/>
```

Comment:
Forgot to mention, this extra layer was adding additional margins from the separator. Since this component is by itself and a "no-label input" it doesn't need a form-row container.